src/clients/Elsa.Api.Client/Resources/WorkflowDefinitions/Enums/PersistenceStrategy.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Elsa.Api.Client.Resources.WorkflowDefinitions.Enums;
+
+/// <summary>
+/// Define the Persistence strategy to store information
+/// </summary>
+public enum PersistenceStrategy

```

Comment:
I am wondering if `LogPersistenceMode` might be a better name. It is more focussed at logging, and "mode" is an alternative to "strategy" - we already use "strategy" for types that implement the strategy pattern. "Persistence" on its own is too broad.

What do you think?

Reply:
Yes , agree ! I'll do the change

Reply:
Change done , using the two commit, I also rename the json key. 
I'll update the Client code once this PR will be done.